main.tex

```diff
@@ -373,22 +367,26 @@ \subsection*{Genetic backgrounds carrying resistance alleles}
 %
 This structure is consistent with an initial introgression of the ancestral F1 haplotype, followed later by introgressions of haplotypes carrying secondary mutations.
 %
-The haplotype networks also illustrate the constrasting levels of non-synonymous variation between \texttt{L995F} and \texttt{L995S} (Figure \ref{fig:networks}).
+The haplotype network also illustrates the constrasting levels of non-synonymous variation between \texttt{L995F} and \texttt{L995S}.
 %
-Two non-synonymous variants are present within the \texttt{L995S} networks, but both are at low frequency, and thus may be neutral or mildly deleterious variants that are hitch-hiking on selective sweeps for the \texttt{L995S} allele.
+Only two non-synonymous variants are present within the \texttt{L995S} groups, and both are at low frequency, thus may be neutral or mildly deleterious variants that are hitch-hiking on selective sweeps for the \texttt{L995S} allele.
 %%
 
 %%
 %
-As well as being found in both species, F1 haplotypes were present in mosquitoes sampled from 4 different countries (Guinea, Burkina Faso, Cameroon, Angola) (Figure \ref{fig:map}).
+The F1 group contained haplotypes from mosquitoes of both species, and from mosquitoes sampled in 4 different countries (Guinea, Burkina Faso, Cameroon, Angola) (Figure \ref{fig:map}).
+%
+The F4, F5 and S2 groups each contained haplotypes from both Cameroon and Gabon.
+%
+The S3 group contained haplotypes from both Uganda and Kenya.
 %
-The F4, F5 and S2 haplotypes were each found in both Cameroon and Gabon.
+The haplotypes within each of these groups were nearly identical across the entire span of the \textit{Vgsc} gene ($\pi < 5.1 \times 10^{-5}\ bp^{-1}$).
 %
-S3 haplotypes were present in both Uganda and Kenya.
+In contrast, diversity among wild-type haplotypes was two orders of magnitude greater (Cameroon \textit{An. gambiae} $\pi = 1.4 \times 10^{-3}\ bp^{-1}$; Guinea-Bissau $\pi = 5.7 \times 10^{-3}\ bp^{-1}$).
```

Comment:
Have added some numbers here to illustrate the diversity contrast between selected and non-selected. I.e., expecting to find a group of haplotypes as closely related by chance is extremely unlikely.

Reply:
I like it, nice.